### PR TITLE
improvements for release: critical billing fixes, templates & copy

### PR DESCRIPTION
**Pull request by slajax**
# tasks

- [x] Fix billing environment variables so CLI works in development mode
- [x] Add error handling to deal with critical billing API failures
- [x] improve copy across the entire CLI to match new positioning
- [x] update templates to have minimal sets & new defaults

### CODE REVIEW

File: src/commands/remove.ts [hunk 1]
```diff
@@ -130,6 +130,19 @@ export default class Remove extends Command {
     return { ...inputs, deleteDescription }
   }
 
+  validateBilling = async (inputs: RemoveInputs): Promise<RemoveInputs> => {
+    let billing = await this.services.billingService.validateBilling(
```
Comment (lucamaraschi) [inline src/commands/remove.ts:134]: This should be `const` and not `let`

File: src/commands/start.ts [hunk 2]
```diff
@@ -119,7 +119,14 @@ export default class Start extends Command {
       )
       // We will only have a non-null serviceUrl if we've started a service
       if (runDetails.serviceUrl) {
-        let startMessage = `Success, service ${name} has been started.\nView the logs here: ${logsURL}\nAccess your service at ${runDetails.serviceUrl}`
+        let full_name = this.ux.colors.successGreen(`${op.name}:${op.version}`)
```
Comment (lucamaraschi) [inline src/commands/start.ts:122]: `let` should be `const`

File: src/commands/status.ts [hunk 3]
```diff
@@ -44,27 +43,34 @@ export default class Status extends Command {
...
+      head: ['Service', 'Run ID', 'Created'],
+      style: {
+        padding: 0,
+        border: [],
       },
-      { sort: 'name' },
-    )
+    })
+
+    let sortedServices = runningServices.sort((a, b) => {
```
Comment (lucamaraschi) [inline src/commands/status.ts:60]: `let` should be `const`

File: src/utils/time.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+export const timeSince = (date: date) => {
+  var seconds = Math.floor((new Date() - date) / 1000)
```
Comment (lucamaraschi) [inline src/utils/time.ts:2]: `var` should be `const`

File: src/utils/time.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  var interval = seconds / 31536000
```
Comment (lucamaraschi) [inline src/utils/time.ts:3]: `var` should be `let`

File: test/commands/publish.spec.ts [hunk 6]
```diff
@@ -126,10 +126,9 @@ it('should convert pipelines to a publishable workflow', async () => {
     ],
   } as OpPipeline
 
-  let {
-    pipelineWorkflows,
-    pipelineCommands,
-  } = cmd.convertPipelinesToWorkflows([pipeline])
+  let { pipelineWorkflows, pipelineCommands } = cmd.convertPipelinesToWorkflows(
```
Comment (lucamaraschi) [inline test/commands/publish.spec.ts]: `let` should be `const`

**Review by lucamaraschi — CHANGES_REQUESTED** [review]

**Review by slaterb1 — CHANGES_REQUESTED** [review]
There are some typescript errors that need resolving https://github.com/cto-ai/cli/runs/1884068236?check_suite_focus=true

**Comment by slajax** [comment]
@slaterb1 @jbottein1 @lucamaraschi I have fixed the type script errors and resolved the comments.

I noticed that the GH action appears to be publishing to `@cto.ai/ops:1.16.-rc.0` however in this case it should be published to `@cto.ai/ops-rc:1.16.1`. If this action runs and publishes it to the main repo, every user will be prompted to install it, so we must not do that until it's ready to be released. That is why we have `ops-rc` as a separate repo.

To avoid the automated steps making this mistake, I have gone ahead and manually published it with `npm publish`.

**Review by lucamaraschi — APPROVED** [review]

**Comment by slajax** [comment]
@slaterb1 can we get a re-review please.

**Comment by slaterb1** [comment]
Will do shortly

**Review by slaterb1 — APPROVED** [review]

**Pull request merged**